Rettelse PR (#62)
MrLongbottom/P9-Rapport: sections/Abstract.tex

Topic models are used to discover topics within a set of documents.
In this paper, we investigate whether using a topic model can improve existing \gls{ir} methods.

> We employ the probabilistic topic model, \gls{lda}, to automatically extract topics from Danish news articles.

Reviewer: ```suggestion
We employ the probabilistic topic model, \acrlong{lda}, to automatically extract topics from Danish news articles.
```

%How can \gls{pr} be used on a dataset without explicit edges?

> We use \gls{lda}, to construct an adjacency matrix, by measuring the Jensen-Shannon distance between the articles' topic distributions.

Reviewer: ```suggestion
We use \acrlong{lda}, to construct an adjacency matrix, by measuring the Jensen-Shannon distance between the articles' topic distributions.
```

> We test various \gls{ir} methods' ability to find documents related to a query, but also to find documents related to topics expressed in queries.

Reviewer: ```suggestion
We test various \acrlong{ir} methods' ability to find documents related to a query, but also to find documents related to topics expressed in queries.
```

> We evaluate \gls{ir} performance of combinations of \gls{lda}, \gls{tf-idf}, \gls{bm25}, \gls{lm}, and \gls{pr}.

Reviewer: ```suggestion
We evaluate \acrlong{ir} performance of combinations of \acrlong{lda}, \acrlong{tf-idf}, \acrlong{bm25}, \acrlong{lm}, and \acrlong{pr}.
```

> We evaluate \gls{ir} performance of combinations of \gls{lda}, \gls{tf-idf}, \gls{bm25}, \gls{lm}, and \gls{pr}.

Author: ```suggestion
We evaluate \acrlong{ir} performance of combinations of \acrlong{lda}, \acrlong{tf-idf}, \acrlong{bm25}, \acrlong{lm}, and \acrlong{pr}.
Our results show that using \acrlong{pr} can improve performance of most \acrlong{ir} methods.
```
